### DISCUSSION TITLE
记忆上次该域名是否翻译

### DISCUSSION THREAD

**Discussion by actforjason** [immersive-translate/immersive-translate#914] [Ideas]
### 新功能 | New Feature

1. 记忆上次该域名的是否翻译的情况

### 为什么您想要添加这些功能 | Why you want to add these features

1. 永不翻译或总是翻译这两个选项太限制性了，有时一段时间需要保持一种选项，总是切换比较麻烦

### 提议 | Proposal

可以把当前选择——是否翻译该域名写入cookie（记忆），决定后续是否翻译

**Comment by theowenyoung** [comment]
在进阶设置里有一个选项也许可以满足你的需求：

<img width="923" alt="图片" src="https://github.com/immersive-translate/immersive-translate/assets/62473795/fe40be9e-dea0-4680-90b4-b288d9fbf89c">


当你手动翻译了一个网页的时候，自动对于该网站在一定时间内开启临时自动翻译。

**Comment by actforjason** [comment]
这个好像不能临时关闭，比如 在外文网站评论的时候翻译会打乱预览排版，但又不想关闭这个网站的翻译，这个时候快捷键取消翻译并记忆当前状态，在评论完后再快捷键切回翻译可以避免评论多个时来回切换按键。

**Reply by theowenyoung** [reply]
如果是这种需求的话，我觉得就用快捷键来回切换就好了？ 因为反正有缓存，再次翻译的时候也是用缓存，而不是真实的请求。

**Reply by actforjason** [reply]
快捷键切换是临时的吧，如果要回复多个，子页面变化或刷新后又会自动翻译需要再切回原文
